uint32 RED = 0
uint32 ORANGE = 1
uint32 GREEN = 2
uint32 YELLOW = 3
uint32 BLUE = 4

float32 X
float32 Y
float32 Area
uint8 Color